Gemfile

```diff
@@ -2,7 +2,9 @@
 # If you modify this file manually all comments and formatting will be lost.
 
 source "https://rubygems.org"
-gem "logstash-core", "3.0.0.dev", :path => "."
+gem "logstash-core", "3.0.0.dev", :path => "./logstash-core"
+# gem "logstash-core-event", "3.0.0.dev", :path => "./logstash-core-event"
```

Comment:
again, any comments will be deleted after a bin/plugin action, so I don't see a point in adding references to the java implementation. The README.md will describe how to switch